core/src/scene/sceneLoader.cpp

```diff
@@ -73,21 +73,15 @@ void SceneLoader::loadShaderConfig(Node shaders, Style& style, Scene& scene) {
     if (!shaders) { return; }
 
     if (Node extensionsNode = shaders["extensions"]) {
-        std::vector<std::string> extensions;
-        if (extensionsNode.IsScalar()) {
-            extensions.push_back(extensionsNode.as<std::string>());
-        } else if(extensionsNode.IsSequence()) {
-            extensions.reserve(extensionsNode.size());
+        if(extensionsNode.IsSequence()) {
             for (const auto& extNode : extensionsNode) {
-                extensions.push_back(extNode.as<std::string>());
+                char buffer[1000]; //sufficient space
+                auto extName = extNode.as<std::string>();
+                sprintf(buffer, "#ifdef %s\n    #extension %s : enable\n    #define TANGRAM_EXTENSION_%s\n#endif",
```

Comment:
There's no requirement for extensions to be enabled at the top of the shader? I always get confused about that.